export/mlab_export_test.py

```diff
@@ -0,0 +1,492 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit tests for collectd-mlab."""
+
+import errno
+import os
+import StringIO
+import time
+import unittest
+
+# Third-party modules.
+import mock
+
+# Module under test.
+import mlab_export
+
+# Too many public methods. Hard to avoid for unit tests.
+# pylint: disable=R0904
+
+
+def disable_show_options(options):
+  """Sets all show options to False."""
+  options.show_nagios = False
+  options.show_rrdfile = False
+  options.show_rawmetric = False
+  options.show_metric = False
+  options.show_skipped = False
+  return options
+
+
+def enable_show_options(options):
+  """Sets all show options to True."""
+  options.show_nagios = True
+  options.show_rrdfile = True
+  options.show_rawmetric = True
+  options.show_metric = True
+  options.show_skipped = True
+  return options
+
+
+class MlabExport_GlobalTests(unittest.TestCase):
+
+  def setUp(self):
+    self._testdata_dir = os.path.join(
+        os.path.dirname(mlab_export.__file__), 'testdata')
+
+  def testunit_LastTime(self):
+    fake_tstamp = os.path.join(self._testdata_dir, 'fake.tstamp')
+    # Artificially set fake timestamp to one hour ago.
+    one_hour_ago = int(time.time() - 3600)
```

Comment:
No problem. This change replaces uses of time.time() with a static timestamp.

This change also includes the minimum additional changes to bring the tests in sync with mlab_export.py. 7eb66eba03850226e3eacab9eb8c7def00735bf4